src/Symfony/Component/Translation/Resources/bin/translation-status.php

```diff
@@ -159,6 +159,11 @@ function printTitle($title)
 
 function printTable($translations, $verboseOutput)
 {
+    if (0 === count($translations)) {
+        echo 'No translations found';
```

Comment:
shouldn't it say `No missing translations found` ?

Reply:
No, here is what `$translations` looks like with a legit locale:
```
array(1) {
  ["fr"]=>
  array(3) {
    ["total"]=>
    int(3)
    ["translated"]=>
    int(3)
    ["missingKeys"]=>
    array(0) {
    }
  }
}

```